client/views/chat.tpl

```diff
@@ -21,7 +21,7 @@
 		<div class="show-more{{#if messages.length}} show{{/if}}">
 			<button class="show-more-button" data-id="{{id}}">Show older messages</button>
 		</div>
-		<div class="messages"></div>
+		<div class="messages" role="log" aria-live="polite" aria-relevant="additions"></div>
```

Comment:
I think we need to add `aria-atomic="true"`. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

VoiceOver on El Capitan re-reads the whole chat window at every change, which quickly becomes unmanageable. I was hoping `aria-atomic` would fix this, but it does not. @cptvitamin, do you have any insight?

Reply:
From the page you linked:

> aria-atomic is not set ("false" by default) so that only the added or removed users should be spoken and not the entire roster each time.

Reply:
I can imagine how this could happen when `more` messages are loaded. perhaps these aria attributes have to be managed by JS to avoid that?

Reply:
No, it happens with literally every message, starting with the second one in a blank chat.